### PR TITLE
Swatang 10: Implement Add skill feature

### PR DESCRIPTION
**Pull request by thienkieu**
### Description
- Implement new feature to add skill profile 
- Implement unit test for feature component

### ☑️ Definition of Done Checklist
- [x] Keep the application can be launched successfully
- [x] Keep the source code is built successfully
- [x] Keep the unit test can run successfully

### Check the box when resolved these items and provide more information

- [ ] Documentation
- [ ] Localization
- [x] Coding
- [x] Unit Tests
- [ ] Integration Tests
- [ ] System Tests

### 📷 Screenshot / GIF

screen-ui
test1

### CODE REVIEW

**Comment by mwd-luongmai** [comment]
hi @thienkieu , please correct the format of subject of pull quest and the name of branch. I think that we should follow the naming convention which be agreed by other team members. Thanks

**Comment by mwd-luongmai** [comment]
@thienkieu another point, 9 commits are looking like too much to review, could you please group the commits or separate the pull request. Thanks

File: src/app/features/skills/skill-list/skill-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+
+@Component({
+	templateUrl: './skill-list.component.html',
+	styleUrls: [ './skill-list.component.scss' ]
+})
+export class SkillListComponent implements OnInit, OnDestroy {
+	constructor(
```
Comment (mwd-luongmai) [inline src/app/features/skills/skill-list/skill-list.component.ts:8]: should remove the empty functions for clean the code

File: src/app/features/skills/skill-profile/skill-profile.component.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
+<div class="flex-comp-container skill-profile">
+    <mat-card>
+      <mat-card-title class="align-left">Create Skill Profile </mat-card-title>
+      <mat-card-content>
+        <form [formGroup]="skillProfile" (ngSubmit)="onSubmit()">
+          <div class="clear-float">
+            <button (click)="onSubmit()" mat-raised-button [disabled]="loading" color="primary" class="add-skill-level">
+              <mat-icon><i class="material-icons md-dark">save</i></mat-icon>
+              Submit skil
```
Comment (mwd-luongmai) [inline src/app/features/skills/skill-profile/skill-profile.component.html:9]: typo mistake => should be "Submit skill" or only "Submit"

File: src/app/features/skills/skill-routing.module.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import { Routes, RouterModule } from '@angular/router'
+import { SkillProfileComponent } from './skill-profile/skill-profile.component'
+import { SkillListComponent } from './skill-list/skill-list.component'
+
+const routes: Routes = [
+  {
+    path: '', component: SkillListComponent
+  },
+  {
+    path: 'skill-profile/:id', component: SkillProfileComponent
```
Comment (mwd-luongmai) [inline src/app/features/skills/skill-routing.module.ts:11]: look like we do a lot of tasks in this pull requests with multiple components. could you please separate to smaller pull request?

File: src/polyfills.ts [hunk 4]
```diff
@@ -77,7 +77,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone' // Included with Angular CLI.
+import 'zone.js/dist/zone.js' // Included with Angular CLI.
```
Comment (mwd-luongmai) [inline src/polyfills.ts:80]: could you please explain about this updates? thanks

Comment (thienkieu) [reply src/polyfills.ts:80]: This is suggestion at https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html#zonejs to fix issue of ckeditor with unitest

**Comment by thienkieu** [comment]
@mwd-luongmai, @quangloc1993 @1ceb3rg   I closed this PR to fix branch name as @mwd-luongmai comment,
Please review at this PR:  https://github.com/mwd-luongmai/angular8-training-frontend/pull/24

**Pull request closed without merging**